### ISSUE TITLE
Regarding PlantUML Class Diagram default notation

### ISSUE THREAD

**Issue by JeremyLoh** [nus-cs2103-AY1920S2/forum#88]
Dear prof @okkhoy , my team (@AY1920S2-CS2103T-W12-1, @khsc96, @exeexe93, @amoscheong97, @cherweijie) would like to check if the following class diagram syntax provided by plantUML is appropriate (asked at today's tutorial at 12pm). 
From https://plantuml.com/class-diagram, the visibility fields for plantUML utilizes the following icons shown below.
![image](https://user-images.githubusercontent.com/50527804/77510830-9adea680-6eaa-11ea-9a0d-a9da8f323dd7.png)

We have attached a sample class diagram for our project below.
![image](https://user-images.githubusercontent.com/50527804/77510785-800c3200-6eaa-11ea-8f20-02bb24f60967.png)

Thank you!

**Comment by okkhoy** [comment]
No. These are not standardized UML notations. So you need to disable them.
In the same page from where you got your screenshot (just below the last line that you have!) you have a way to disable them.

![image](https://user-images.githubusercontent.com/1023494/77544356-dea1d200-6ee3-11ea-924f-4b81c8356edd.png)

Additionally,  use `hide circle` to hide the class symbol.

**Comment by JeremyLoh** [comment]
Thank you prof @okkhoy for the clarification!
